<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>791</width>
      <height>501</height>
     </rect>
    </property>
    <widget class="QWidget" name="Page1">
     <widget class="QPushButton" name="createProfile">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>420</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Create new Profile</string>
      </property>
     </widget>
     <widget class="QLabel" name="profileLabel">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>10</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>DejaVu Sans</family>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Profiles:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Start">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>420</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
     <widget class="QWidget" name="profiles" native="true">
      <property name="geometry">
       <rect>
        <x>49</x>
        <y>59</y>
        <width>681</width>
        <height>361</height>
       </rect>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>651</width>
         <height>341</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="profileLayout">
        <property name="spacing">
         <number>5</number>
        </property>
       </layout>
      </widget>
     </widget>
     <widget class="QPushButton" name="updateProfile">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>420</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Read Or Update Profile</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteProfile">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>420</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete Profile</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Page2">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>DejaVu Sans</family>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Create a Profile:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ProfileName">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>100</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>100</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>DejaVu Sans</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Profile Name:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>150</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>DejaVu Sans</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Basal Rates:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>DejaVu Sans</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Carbohydrate Ratio:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>250</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>DejaVu Sans</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Correction Factor:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="BasalRates">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>150</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="CorrectionFactor">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>250</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="Carbohydrate">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>200</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="SubmitForm">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>390</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>300</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>DejaVu Sans</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Profile Pin:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Pin">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>310</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page3">
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>90</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>DejaVu Sans</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Profile Name:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>140</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>DejaVu Sans</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Basal Rates:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>190</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>DejaVu Sans</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Carbohydrate Ratio:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>260</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>DejaVu Sans</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Correction Factor:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>330</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>DejaVu Sans</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Profile Pin:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>301</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>DejaVu Sans</family>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Update the Profile:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ProfileName2">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>90</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="BasalRates2">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>140</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="Carbohydrate2">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>190</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="CorrectionFactor2">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>260</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="Pin2">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>330</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="Update">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>400</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Update</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page4">
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>110</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Home Screen</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
